### PR TITLE
Replace GEKKO define with MWCC __PPCGEKKO__

### PR DESCRIPTION
**Pull request by r-burns**

### CODE REVIEW

File: src/Runtime/platform.h [hunk 1]
```diff
@@ -89,6 +89,10 @@ typedef jmp_t jtbl_t[];
 /// A @c void callback with no arguments.
 typedef void (*Event)(void);
 
+#if defined(__PPCGEKKO__)
+#define GEKKO
+#endif
+
```
Comment (ribbanya) [inline src/Runtime/platform.h:95]: Shouldn't we just find/replace instead?

Comment (r-burns) [reply src/Runtime/platform.h:95]: Well it'll need to be defined manually by gcc et al so I think it makes sense to abstract it behind GEKKO.

Comment (ribbanya) [reply src/Runtime/platform.h:95]: Yeah but then they'd need to manually define `GEKKO`, right? So they can just manually define `__PCCGEKKO__` instead.

Comment (r-burns) [reply src/Runtime/platform.h:95]: Ok fair enough, I'll just replace them all.

**Review by ribbanya — APPROVED** [review]

**Pull request merged**